### PR TITLE
Integrate dp-cantabular-csv-exporter dockerfile

### PR DESCRIPTION
**Pull request by andre-urbani**
- Integrate dp-cantabular-csv-exporter dockerfile
- Integrate minio as local emulation of Amazon S3

Trello - https://trello.com/c/SP0Yhuc5/5190-add-local-dockerfile-for-dp-cantabular-csv-exporter-and-integrate-into-dp-compose-cantabular-import-journey

### CODE REVIEW

File: cantabular-import/deps.yml [hunk 1]
```diff
@@ -34,3 +34,10 @@ services:
      - POSTGRES_PASSWORD=mysecretpassword
     ports:
     - 5432:5432
+  minio:
+    image: 'bitnami/minio:latest'
+    ports:
+      - '9000:9000'
+    environment:
+      - MINIO_ACCESS_KEY=minio-access-key
+      - MINIO_SECRET_KEY=minio-secret-key
```
Comment (redhug1) [inline cantabular-import/deps.yml:43]: Needs new line at end

File: cantabular-import/dp-cantabular-csv-exporter.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        depends_on:
+            - kafka
+        ports:
+            - 26300:26300
+        environment:
+            KAFKA_ADDR:         "kafka:9092"
+            DATASET_API_URL:    "http://dp-dataset-api:22000"
+            RECIPE_API_URL:     "http://dp-recipe-api:22300"
+            CANTABULAR_URL:     "http://dp-cantabular-server:8491"
+            SERVICE_AUTH_TOKEN: $SERVICE_AUTH_TOKEN
```
Comment (redhug1) [inline cantabular-import/dp-cantabular-csv-exporter.yml:24]: Needs new line at end

File: cantabular-import/dp-cantabular-csv-exporter.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            - -c
+            - ./reflex
+        volumes:
+            - ../../dp-cantabular-csv-exporter:/dp-cantabular-csv-exporter
+        depends_on:
+            - kafka
+        ports:
+            - 26300:26300
+        environment:
+            KAFKA_ADDR:         "kafka:9092"
```
Comment (redhug1) [inline cantabular-import/dp-cantabular-csv-exporter.yml:20]: Is this needed here:
 BIND_ADDR:          ":26300"

(ask Joshua or David)

**Review by redhug1 — CHANGES_REQUESTED** [review]
A few minor things to fix

File: cantabular-import/dp-cantabular-csv-exporter.yml (hunk 3 above)
Comment (andre-urbani) [reply cantabular-import/dp-cantabular-csv-exporter.yml:21]: Yes, I just checked the other services that are part of the cantabular import process and they both include the BIND_ADDR variable, so I've added it in

**Review by redhug1 — APPROVED** [review]

**Pull request merged**